visualizations/fan_chart/webviz_fan_chart/__init__.py

```diff
@@ -127,10 +130,20 @@ def make_marker(obs, color):
     }
 
 
+def index_observations(obs):
+    obs.set_index(
+        obs['index'],
+        drop=True,
+        inplace=True
+    )
+    return obs
```

Comment:
There is no need for a return value here. I think it is more readable to inline this function.

Reply:
@eivindjahren the reason was actually to be able to properly test that the index gets set correctly every time. I found it easier to just extract it into a function, which also reduced a couple of lines of code.